src/components/BalanceGame/AnimatedText.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import { AnimatePresence, motion } from "framer-motion";
+
+interface AnimatedTextProps {
+  children: React.ReactNode;
+  // 애니메이션 트리거 값 (예: 클릭한 옵션 여부)
+  animateTrigger: boolean;
```

Comment:
```suggestion
  isAnimateTrigger: boolean;
```
`animateTrigger`라는 이름만 봤을 때는 boolean 타입인지 유추하기 어려울 수도 있을 것 같아요🤔